components/homme/src/theta-l/share/element_ops.F90

```diff
@@ -154,8 +154,11 @@ subroutine get_field_i(elem,name,field,hvcoord,nt)
          field = elem%state%w_i(:,:,1:nlevp,nt)
       endif
     case('geo_i');
-      !is phinh_i set for HY runs or should here be an abort?
-      field = elem%state%phinh_i(:,:,1:nlevp,nt)
+      if(theta_hydrostatic_mode) then
+          call abortmp('ERROR: get_field_i is not supported for phi_i in theta HY')
+      else
+          field = elem%state%phinh_i(:,:,1:nlevp,nt)
+      endif
```

Comment:
since we are modifying this file, might as well fix it instead of aborting.  Something like:

    if(theta_hydrostatic_mode) then
       call phi_from_eos(hvcoord,elem%state%phis,elem%state%vtheta_dp(:,:,:,nt),elem%state%dp3d(:,:,:,nt),field)
    else
       field = elem%state%phinh_i(:,:,:,nt)
    endif

Reply:
done, thanks. ran HY output and all looks ok.